src/03-modality-agnostic-files.md

```diff
@@ -162,16 +162,20 @@ participants.json
 ```
 
 The purpose of this RECOMMENDED file is to describe properties of participants
-such as age, sex, handedness.
+such as age, sex, handedness, species, strain, strain_rrid, diagnosis.
 If this file exists, it MUST contain the column `participant_id`,
 which MUST consist of `sub-<label>` values identifying one row for each participant,
 followed by a list of optional columns describing participants.
 Each participant MUST be described by one and only one row.
 
-Commonly used *optional* columns in `participant.tsv` files are `age`, `sex`,
-and `handedness`. We RECOMMEND to make use of these columns, and
-in case that you do use them, we RECOMMEND to use the following values
-for them:
+When different from `homo sapiens`, `participants.tsv` SHOULD include a `species`
+column, and the value MUST be the string of the binomial species name from
+[NCBI Taxonomy](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi).
+
+Commonly used *optional* columns in `participants.tsv` files are `age`, `sex`,
+`handedness`, `strain`, `strain_rrid` and `diagnosis`. We RECOMMEND to make use
```

Comment:
Could group be added to this list as its used in the example below?

Reply:
Good question, I'm not sure we should because `group` is also used below to illustrate how to describe an additional column that is not part of the optional ones in the `participants.json` example. From my understanding, `group` may have different meanings depending on the study and it would be hard to define it with a general example